Sources/TokamakCore/Renderer.swift

```diff
@@ -31,9 +31,6 @@ public protocol Renderer: AnyObject {
    */
   associatedtype TargetType: Target
 
-  /// Reconciler instance used by this renderer.
-  var reconciler: StackReconciler<Self>? { get }
```

Comment:
It wasn't referenced anywhere, so no need to have it as a requirement. It also allowed making this property completely `private` in `Renderer` implementations.